create-tables.sql

```diff
@@ -0,0 +1,22 @@
+BEGIN; CREATE TABLE public.projects (
+    "name" varchar(100) NOT NULL DEFAULT ''::character varying,
+    version_id int4 NOT NULL DEFAULT 0,
+    last_accessed timestamptz NULL,
+    CONSTRAINT projects_pkey PRIMARY KEY (name)
+  );
+  CREATE INDEX projects_index_last_accessed
+    ON public.projects
+    USING btree
+    (last_accessed);
+
+  CREATE TABLE public.url_index_store (
+    project_name varchar(100) NOT NULL DEFAULT ''::character varying,
+    url varchar(2000) NOT NULL,
+    "path" varchar(2000) NOT NULL,
+    CONSTRAINT url_index_store_pkey PRIMARY KEY (project_name, url)
```

Comment:
Is there a reason not to keep the foreign key? I think it would be better to have one (ideally with cascading-deletes) if that's a possibility. Would save us from future cruft.

Reply:
The foreign key won't actually work, because we add to the child table before the corresponding row exists in the parent table. Plus, the exiting data will be inconsistent anyway. 

I think we could re-add the constraint after we have migrated to postgres, moved to row locking, and cleaned up the data.

Reply:
(For clarity, this foreign key relation has never actually worked in production. It's a phantom 👻 )

Reply:
> we add to the child table before the corresponding row exists in the parent table

Right, good point. I think we'll be able to re-add this in the next PR which adds the locking though? In which case, might it be a good idea to do that there?

Reply:
Yup, I think we can do this with the row-lock PR :+1: